client/sections.js

```diff
@@ -360,7 +360,7 @@ const sections = [
 	},
 	{
 		name: 'reader',
-		paths: [ '/read/search', '/recommendations' ],
+		paths: [ '/read/search', '/([a-z]{2,3}|[a-z]{2}-[a-z]{2})/read/search', '/recommendations' ],
```

Comment:
Just curious, what are the 3 letter language codes that can be used?

Reply:
It looks like we have a few, "als" for Alemannisch (the language of the Amish)  https://github.com/Automattic/wp-calypso/blob/trunk/packages/languages/src/languages-meta.json#L12